microsoftgraph/microsoft-graph-explorer-v4
# Enhancement: sample queries cache

## Overview

Fixes #2119 

### Notes

When a successful call is made to get samples, they are cached for use in case their next run experiences a failure.
This ensures the cache has the last known good set of samples.

The older version of cached sample queries is used as a fallback of the fallback. This will eventually be phased out so that we can remove it from the build.

## Testing Instructions

* Load Graph Explorer in the local environment
* Change the base url for the samples url so that it once the app loads, it receives an error 404
* Notice that a cached version exists, with all the previously received samples
